examples/daisyworld.jl

```diff
@@ -304,6 +336,4 @@ plot(p1, p2, p3, layout = (3, 1), size = (500, 800))
 #
 # Finally, as the sun fades back to normal levels, both the temperature and white daisy
 # population struggle to find equilibrium. The counterbalancing force of the black daisies
```

Comment:
This is no longer true with the scenario you have generated. This text refers to a daisyworld there the solar luminosity starts at 0.75 and it's temperature never exceeds around 40 degrees. 

Will need to either change this text to refer to the new plot, or change the parameters back.